src/mc_bench/apps/api/routers/comparison.py

```diff
@@ -60,8 +60,15 @@ def get_comparison_batch(
             headers={"WWW-Authenticate": "Bearer"},
         )
 
+    test_set_filter = ""
```

Comment:
We will not implement this as a filter in get_comparison. Please do not update this endpoint yet.